validator/utils.py

```diff
@@ -193,4 +207,15 @@ def get_fixed_data(data_dir):
             ]
         )
 
-    return df_innovation, df_domain, df_questions
+    # Now load the feature_weights and default feature_weight_id, if found.
+    try:
+        with open(os.path.join(data_dir, "feature_weights.json")) as f:
+            feature_weights = json.load(f, object_pairs_hook=OrderedDict)
+            default_feature_weights_id = "d3732be6-a759-43aa-9e1a-3e9bd94f8b6b"
```

Comment:
No need for default_fw_id here or at line 218 - since they are in feature_weights.json, right?

Reply:
correct. And don't return it seperately at all